Scripts/projView.js

```diff
@@ -1,33 +1,43 @@
-var projControl = {};
+(function(module){
 
 
-projControl.navDisplayToggle = function() {
-  $('.globalNav').on('click', 'li', function() {
-    console.log('navToggle');
-    $('.tab-content').hide();
-    console.log($(this).data('content'));
-    $('#' + $(this).data('content')).show();
-  });
+  var projControl = {};
+  var booksView = {};
 
-  $('.globalNav .tab:first').click();
-};
 
-projControl.navBurgerShow = function() {
-  $('.icon-menu').on('click', function() {
-    $('nav ul').toggle();
-  });
-};
+  projControl.navDisplayToggle = function() {
+    $('.globalNav').on('click', 'li', function() {
+      $('.tab-content').hide();
+      $('#' + $(this).data('content')).show();
+    });
+
```

Comment:
make sure with your nav function that you have a data-content on the nav item that matches up to the id of the section you want to show.